public/index.html

```diff
@@ -5,17 +5,59 @@
         <title>Chat</title>
         <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.1/angular.min.js"></script>
         <script src="js/main.js"></script>
+        <script src="js/chatService.js"></script>
+        <script src="js/ConversationController.js"></script>
+        <script src="js/UserListController.js"></script>
+        <script src="js/ConversationListController.js"></script>
     </head>
     <body ng-controller="ChatController">
         <a ng-show="!loggedIn" href="{{loginUri}}">Log in</a>
         <section ng-show="loggedIn">
-            <p>Logged in as {{user.name}}.</p>
-            <h2>List of registered users</h2>
-            <ul>
-                <li ng-repeat="user in users">
-                    {{user.name}}
-                </li>
-            </ul>
+                <p>Logged in as {{user.name}}.</p>
+                <div ng-controller="UserListController" ng-if="user._id">
```

Comment:
I think you'll want to split the sections under sub controllers into seperate templates